app/controllers/api/v2/trends_controller.rb

```diff
@@ -1,51 +1,46 @@
 module Api
   module V2
+    # This controller just informs about the plugin
     class TrendsController < V2::BaseController
-      include Foreman::Controller::Parameters::Trend
-
-      before_action :find_resource, :only => [:show, :destroy]
+      prepend_before_action :fail_and_inform_about_plugin
 
       TRENDABLE_TYPES = [
         'Environment', 'Operatingsystem', 'Model', 'FactName', 'Hostgroup',
         'ComputeResource'
       ].freeze
```

Comment:
I think we could also drop these and the param descriptions here

Reply:
Well I wanted to keep the apidocs intact :thinking:
But we cat keep just the methods and descriptions, parameters are not strictly necessary to keep around, dropping them :+1: